src/editor/ui_menus/main_menu_map_options.cc

```diff
@@ -316,10 +327,23 @@ MainMenuMapOptions::MainMenuMapOptions(EditorInteractive& parent, Registry& regi
 	add_tag_checkbox(&tags_box_, "3teams", localize_tag("3teams"));
 	add_tag_checkbox(&tags_box_, "4teams", localize_tag("4teams"));
 
-	balancing_dropdown_.set_autoexpand_display_button();
 	balancing_dropdown_.add(localize_tag("balanced"), "balanced");
 	balancing_dropdown_.add(localize_tag("unbalanced"), "unbalanced");
-	tags_box_.add(&balancing_dropdown_);
+	tags_box_.add(&balancing_dropdown_, UI::Box::Resizing::kFullSize);
+	tags_box_.add_space(padding_);
+
+	theme_dropdown_.add(_("(none)"), "");
+	for (const std::string& theme :
+	     g_fs->list_directory(std::string(kTemplateDir) + "loadscreens/gameloading/")) {
+		const std::string name(g_fs->fs_filename(theme.c_str()));
+		// TODO(Nordfriese): There must be a better way to localize the theme names
+		std::string localized_name =
```

Comment:
We should also have a desert theme, so we can map the old world names to this.

For localizing, you could stick them in a map, might look cleaner.

Reply:
+1 for a desert theme, but I don't have any images that could be interpreted, even with a lot of good will, as desert-themed…
You're welcome to upload some if you have any though :)

Reply:
We still need to support it in the code, even if we don't have any yet.